src/zoomifytilesource.js

```diff
@@ -0,0 +1,113 @@
+(function($) {
+
+    /**
+     * @class ZoomifyTileSource
+     * @classdesc A tilesource implementation for the zoomify format.
+     *
```

Comment:
@foobarable Can you add the link to the format definition and a note that we don't support the XML version here?